Big2/src/components/tabscreens/Rules.js

```diff
@@ -61,7 +61,7 @@ In most popular variations, ending with a single or double two is not allowed.
 </Text>
 
 <Button title="Go to Home"
-        onPress={() => navigation.navigate('Home')}> 
+        onPress={() => navigation.navigate(Home)}> 
 </Button>
```

Comment:
Might not be necessary because we have a back button on the Header of this Screen, but we might change this button to be a Play Now